pyartcd/requirements.txt

```diff
@@ -11,6 +11,7 @@ Jinja2
 jira >= 3.4.1  # https://github.com/pycontribs/jira/issues/1486
 openshift-client
 pygit2 == 1.10.1 # https://github.com/libgit2/pygit2/issues/1176
+PyGithub >= 1.57
```

Comment:
pyartcd uses ghpai as the GitHub API client.

Reply:
I want to use a more straightforward method instead of pulling the file, making changes, then pushing.

Reply:
I mean use ghapi is recommended instead of PyGithub.

Reply:
ghapi is valuable for cloning data from remote to local env and pushing data to remote, but I think we can simplify the workflow as the pr did

Reply:
I think ghapi is capable of more than file operations on repositiories. I prefer to stick to a single dependency in the same area if possible. Have not seen a good rationale why have two deps would give a better solution. Could you investigate again?

Reply:
No, ghapi is a client library for GitHub API. It doesn't clone a git repo.